pkg/psalabelsyncer/podsecurity_label_sync_controller_test.go

```diff
@@ -361,6 +365,26 @@ func TestPodSecurityAdmissionLabelSynchronizationController_sync(t *testing.T) {
 			expectNSUpdate:   true,
 			expectedPSaLevel: "restricted",
 		},
+		{
+			name:   "SA with restricted SCC assigned in am NS with previous enforce labels",
```

Comment:
What does "am NS" mean? Do you mean "a NS"?

Reply:
"an NS" - it's a typo